require.js

```diff
@@ -1031,16 +1028,18 @@
                 config = locationLoader.config,
                 extension = Require.extension(id),
                 location, result;
-            if (!extension || (
-                extension !== "js" &&
-                extension !== "json" &&
-                config.moduleTypes.indexOf(extension) === -1
-            )) {
+            if (
+                !extension || (
+                    extension !== "js" &&
+                        extension !== "json" &&
+                            config.moduleTypes.indexOf(extension) === -1
+                    )
```

Comment:
I would suggest:

```
if (
    !extension || (
        extension != "js" &&
        extension != "json" &&
        config.modulesTypes.indexOf(extension) === -1
    )
) {
}
```

…to keep associative terms on the same indentation level and per K&R style match the close bracket to the indent level of the line that introduced the open bracket.